aim/cli/up/utils.py

```diff
@@ -27,3 +32,112 @@ def build_uvicorn_command(host, port, num_workers, ssl_keyfile, ssl_certfile):
+            tf_logs_cache[path] = {}
+
+        run_cache = tf_logs_cache[path]
+        if run_cache:
+            run = Run(repo=repo_inst, run_hash=run_cache['run_hash'])
+        else:
+            run = Run(repo=repo_inst)
+            run['tf_logdir'] = path
+            run_cache.update({
+                'run_hash': run.hash,
+                'events': {},
+            })
+        run_tf_events = run_cache['events']
+
+        events_to_process = []
+        events = [
+            os.path.join(root, file)
+            for root, dirs, files in os.walk(path)
+            for file in files if file.endswith('v2')
+        ]
+
+        for event in events:
+            last_modified_at = os.path.getmtime(event)
+            try:
+                assert last_modified_at == run_tf_events[event]['last_modified_at']
+            except (KeyError, AssertionError):
+                # Something has changed or hasn't been processed before
+                events_to_process.append(event)
+                try:
+                    run_tf_events[event]['last_modified_at'] = last_modified_at
+                except KeyError:
+                    # Completely new event
+                    run_tf_events[event] = {
+                        'last_modified_at': last_modified_at,
+                        'values': {},
+                    }
+
+        for event_file in events_to_process:
+            run_tf_event = run_tf_events[event_file]
+            for event in summary_iterator(event_file):
+                timestamp = event.wall_time
+                step = event.step
+                for value in event.summary.value:
+                    tag = value.tag
+                    plugin_name = value.metadata.plugin_data.plugin_name
+                    value_id = f'{tag}_{plugin_name}'
+                    if (
+                        value_id in run_tf_event['values'] and
+                        run_tf_event['values'][value_id]['timestamp'] >= timestamp
+                    ):
+                        # prevent previously tracked data from re-tracking upon file update
+                        continue
+                    if plugin_name == 'images':
+                        tf_tensors = value.tensor.string_val[2:]
+                        track_val = [
+                            Image(tf.image.decode_image(tf_tensor).numpy()) for tf_tensor in tf_tensors
+                        ]
+                        if len(track_val) == 1:
+                            track_val = track_val[0]
+                    elif plugin_name == 'scalars':
+                        track_val = value.simple_value
+                    else:
+                        # unsupported plugin type
+                        continue
+                    run_tf_event['values'][value_id] = {
+                        'step': step,
+                        'timestamp': timestamp
+                    }
+                    run.track(track_val, name=tag, step=step)
```

Comment:
lets use `run._track_impl()` in order to pass `event.wall_time`. otherwise we will log the time roughly about `aim up --tf_logs` was ran. 
the similar case is when we convert aim v2 logs (https://github.com/aimhubio/aim/blob/9ba5fdca36e69ab6cbb74296333c2f8bc56392a4/aim/cli/upgrade/utils.py#L75)